src/main/java/seedu/mtracker/console/InputParser.java

```diff
@@ -184,9 +184,12 @@ public void getIndexNumber(String[] commandComponents) {
         }
     }
 
-    public void getSearchString(String[] commandComponents) {
+    public void constructSearchString(String[] commandComponents) {
         try {
-            searchString = commandComponents[SEARCH_STR_INDEX];
+            searchString = commandComponents[SEARCH_STR_INDEX_START];
+            for (int i = SEARCH_STR_INDEX_START + 1; i < commandComponents.length; i++) {
+                searchString += SEPARATOR + commandComponents[i];
+            }

```

Comment:
I dont think this addresses what the user is pointing. 
Say user input: find   (with some spaces in between like the user screenshot)      phrase to find
^ this implementationn would give " (with those spaces)  phrase to find" as the search string. So there is that front space. Are we going to remove the extra spaces? If so u just need to do searchString.trim()

Reply:
We're not trimming. Just searching the whole string. :)

Reply:
Alright!